src/main/scala/encry/view/history/processors/BlockHeaderProcessor.scala

```diff
@@ -161,7 +161,7 @@ trait BlockHeaderProcessor extends Logging { //scalastyle:ignore
         val score: Difficulty = Difficulty @@ (parentScore + difficulty)
         val betterScore: Boolean = bestHeaderIdOpt
           .flatMap(scoreOf)
-          .exists(_ < score)
+          .exists(_ <= score)
```

Comment:
To be deemed better `Header.score` should be strictly greater than score of the header it is compared with